enterprise/releases/2024-10-0.mdx

```diff
@@ -0,0 +1,92 @@
+- This release removes the use of the **Object storage**
+  - With the new packed storage format, pganalyze Enterprise Server now stores all data within the main database, including log texts
+  - This potentially affects database storage size, although the pganalyze team has actually observed storage reductions due to the packed format on our cloud-based environment, despite the extra data being stored
+  - Additionally we recommend reviewing your encryption configuration for your database server, since pganalyze now relies on the database storage to be encrypted for security of log texts (instead of a separate log encryption key)
+  - After upgrading the installation will still be able to read from the object storage for old logs, but no longer write to it. After 7 days have passed (the log retention time) you can remove all object storage related configuration.
+
+## Features
+
+- Index Advisor
+  - Add support for "pg_bigm", "cube" and "earthdistance" extensions <!-- 3953 -->
+  - Avoid Internal Error for tables that had no scans/query activity in the last 7 days <!-- 3956 -->
+  - Avoid errors when unsupported index types/data types are referenced by a table <!-- 3954 -->
+  - Add support for "ltree" extension <!-- 3957 -->
+  - Add ability to filter out acknowledged state insights <!--3943  -->
+  - Fix the count in "QUERIES" column in insights list <!-- 4288 -->
+    - In the Missing Indexes tab in the Index Advisor page, the column "QUERIES" was showing the number of missing indexes in that insight (table) previously. Now this column shows the number of unique affected queries due to these missing indexes as intended.
+  - Show affected index count of each table in insights list <!-- 4289 -->
+  - Recommend removing unused expression indexes in Unused Index alert <!-- 4185 -->
+    - Previously expression indexes were excluded because they produce column statistics that the query planner can benefit from. But this was confusing for users who knew some indexes weren't being queried, and yet we didn't recommend deleting them. You may want to additionally `CREATE STATISTICS` for the expression when dropping an expression index if you notice worse query plans.
+- Automated EXPLAIN and EXPLAIN Plans
+  - Automated EXPLAIN: Add a better support for handling new lines with the text log format <!-- 3937 -->
+  - View EXPLAIN Source: Hide "Async Capable" field when converting from JSON to text format <!-- 3946 -->
+  - Support all Aggregate nodes with text format explain <!-- 4020 -->
+    - Previously, only the plain Aggregate with text format explain was supported. With this change, we support GroupAggregate, HashAggregate, and MixedAggregate better.
+  - Revise EXPLAIN details layout <!-- 4174 -->
+- Update bundled collector from 0.58.0 to 0.61.0 <!-- XXX -->
+  - See the [collector changelog](XXX) for the full details
```

Comment:
I missed that comment somehow - I think the memory allocation is a bit of a more complex situation with the bundled collector, since I think its actually other parts of the app that are more memory intensive (and I've not heard of any collector OOMs of the combined collector). I'll leave as is for now.